src/main/kotlin/ffc.airsync.api/services/org/OrgResource.kt

```diff
@@ -84,7 +86,7 @@ class OrgResource {
     @Path("/$ORGIDTYPE")
     @RolesAllowed("ORG", "ADMIN")
     fun remove(@PathParam("orgId") orgId: String): Response {
-        printDebug("Remove org $orgId")
+        logger.info("Remove organization Id: $orgId")
```

Comment:
ควร Log user ที่ลบด้วย
```suggestion
        logger.info("Remove organization [$orgId] by user [$userId]")
```